src/app_core.py

```diff
@@ -123,6 +123,13 @@ def strdef(name, default):
 app.config["CAPTCHA_SECRETKEY"] = os.getenv("CAPTCHA_SECRETKEY", None)
 app.config["CAPTCHA_SITEKEY"] = os.getenv("CAPTCHA_SITEKEY", None)
 
+app.config["THRESHOLD_TOTAL_SPENDABLE"] = int(os.getenv("THRESHOLD_TOTAL_SPENDABLE"))
+app.config["THRESHOLD_TOTAL_RECEIVABLE"] = int(os.getenv("THRESHOLD_TOTAL_RECEIVABLE"))
+app.config["THRESHOLD_TOTAL_OCHAIN_BAL"] = int(os.getenv("THRESHOLD_TOTAL_ONCHAIN_BAL"))
+app.config["THRESHOLD_AVAIL_BTC_EXCH_BAL"] = float(os.getenv("THRESHOLD_AVAIL_BTC_EXCH_BAL"))
+app.config["THRESHOLD_AVAIL_NZD_EXCH_BAL"] = float(os.getenv("THRESHOLD_AVAIL_NZD_EXCH_BAL"))
+app.config["THRESHOLD_AVAIL_NZD_BANK_BAL"] = float(os.getenv("THRESHOLD_AVAIL_NZD_BANK_BAL"))
```

Comment:
```suggestion
app.config["LN_SPENDABLE_MIN"] = int(os.getenv("LN_SPENDABLE_MIN", 1000000))
app.config["LN_RECEIVABLE_MIN"] = int(os.getenv("LN_RECEIVABLE_MIN", 1000000))
app.config["LN_ONCHAIN_MIN"] = int(os.getenv("LN_ONCHAIN_MIN", 3000000))
app.config["EXCHANGE_MIN"] = json.loads(os.getenv("EXCHANGE_MIN", '{"BTC": 0.05, "NZD": 2000}'))
app.config["BANK_MIN"] = json.loads(os.getenv("BANK_MIN", '{"NZD": 2000}'))
```

update monitor_endpoint.py to use these more descriptive and concise variables

Reply:
making changes here as well.